tests/app/authentication/test_jwe.py

```diff
@@ -10,43 +9,37 @@
 
 class JWETest(unittest.TestCase):
     def setUp(self):
-        settings.EQ_USER_AUTHENTICATION_SR_PRIVATE_KEY = TEST_DO_NOT_USE_SR_PRIVATE_PEM
-        settings.EQ_USER_AUTHENTICATION_RRM_PUBLIC_KEY = TEST_DO_NOT_USE_RRM_PUBLIC_PEM
-        settings.EQ_USER_AUTHENTICATION_SR_PRIVATE_KEY_PASSWORD = "digitaleq"
+        self.decryptor_args = (
+            TEST_DO_NOT_USE_SR_PRIVATE_PEM,
+            'digitaleq',
```

Comment:
Not sure why this was originally declared directly here; might make sense to move it into a 'TEST_DO_NOT_USE_SR_PRIVATE_PEM_PASSWORD'?